### ISSUE TITLE
FTS support?

### ISSUE THREAD

**Issue by AlexErrant** [electric-sql/pglite#179]
How's support for FTS?

On https://pglite.dev/repl/ running `SELECT 'a fat cat sat on a mat and ate a fat rat'::tsvector @@ 'cat & rat'::tsquery;` works, which is cool.

However, running `SELECT to_tsvector('fat cats ate fat rats') @@ to_tsquery('fat & rat');` fails with `y is not a function`. Then trying to run the first query yields `null`, which is weird.

Queries come from https://www.postgresql.org/docs/current/textsearch-intro.html

Stacktrace:

```
Uncaught (in promise) RuntimeError: memory access out of bounds
    at postgres.DSvoAWnW.wasm:0x1eea49
    at postgres.DSvoAWnW.wasm:0x326c39
    at postgres.DSvoAWnW.wasm:0x332072
    at invoke_vi (index.Cg2RIL2X.js:3289:142550)
    at postgres.DSvoAWnW.wasm:0x32fc89
    at FM.execProtocolRaw (index.Cg2RIL2X.js:3289:204501)
    at FM.execProtocol (index.Cg2RIL2X.js:3289:204698)
    at FM.$ (index.Cg2RIL2X.js:3289:210185)
    at index.Cg2RIL2X.js:3289:209134
```

**Comment by gregnr** [comment]
Not sure if this helps debug the issue, but I noticed that the `'simple'` config does work:
```sql
SELECT to_tsvector('simple', 'test');
```

**Comment by samwillis** [comment]
This is fixed in the just published v0.2.2
